python/cudf/cudf/_lib/parquet.pyx

```diff
@@ -124,6 +124,7 @@ def _parse_metadata(meta):
 
 
 cpdef read_parquet(filepaths_or_buffers, columns=None, row_groups=None,
+                   use_arrow_schema=True,
                    use_pandas_metadata=True,
                    Expression filters=None):
```

Comment:
```suggestion
                   use_pandas_metadata=True,
                   Expression filters=None,
                   use_arrow_schema=True):
```
Let's add the new parameters towards the end, so that there is no breakage in calling this method.